metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -238,6 +281,35 @@ void main() {
       },
     );
 
+    test(
+      ".pop() updates the current page parameters",
```

Comment:
✅ 
I believe, in this test, we should check whether the `pageParameters` change when the `pop` method is called. It means we should get the current state of the `pageParameters`, call the `pop` method, and then expect that the `pageParameters` changed. What do you think?